app/static/js/connection.js

```diff
@@ -67,16 +67,19 @@ function updateUsers() {
 }
 
 function headers(xhr) {
-    xhr.setRequestHeader ("Authorization", "Token " + TOKEN);
+    xhr.setRequestHeader("Authorization", "Token " + TOKEN);
 }
 
 $(document).ready(() => {
     let uri = location.protocol + '//' + document.domain + ':' + location.port + "/api/v2";
     socket = io.connect(location.protocol + '//' + document.domain + ':' + location.port);
 
-    socket.on("pong", (data) => {
-        $("#ping").text(data);
-    });
+    setInterval(() => {
+        const start = Date.now();
+        socket.volatile.emit("ping", () => {
+            $("#ping").text(Date.now() - start);
+        });
+    }, 5000);
```

Comment:
🏓

Reply:
What's that? :smile:

Reply:
`:ping_pong:`

Reply:
Oh, my phone could not recognize that emoji 😄